### PR TITLE
Disallow empty import statements

### LINKED ISSUE #5866: Empty import statement crashes (ossfuzz#12670)
Found by ossfuzz (12670)

```
import "";
```

Causes an internal error when used via standard json **without** a read callback.

```
{
  "component" : "general",
  "formattedMessage" : "InternalCompilerError: \n",
  "message" : "Internal compiler error (/Users/alex/Projects-Changes-ny/solidity/libsolidity/interface/CompilerStack.cpp:770):",
  "severity" : "error",
  "type" : "InternalCompilerError"
}
```

### PR DESCRIPTION
**Pull request by axic**
Fixes #5866.

### CODE REVIEW

**Comment by axic** [comment]
This is a fix I think we should have, though should also check why it failed with the assertion `!path.empty()` in `resolveImports`, e.g. why wasn't the `File not initially supplied` message resulting in the same.

File: libsolidity/interface/CompilerStack.cpp [hunk 1]
```diff
@@ -681,6 +681,15 @@ StringMap CompilerStack::loadMissingSources(SourceUnit const& _ast, std::string
 	for (auto const& node: _ast.nodes())
 		if (ImportDirective const* import = dynamic_cast<ImportDirective*>(node.get()))
 		{
+			if (import->path().empty())
```
Comment (chriseth) [inline libsolidity/interface/CompilerStack.cpp]: I have the feeling that the error should be reported by the parser and we should only have an assertion here.

Comment (axic) [reply libsolidity/interface/CompilerStack.cpp]: Changed.

**Review by erak — DISMISSED** [review]

**Review by chriseth — DISMISSED** [review]
Please rebase.

**Comment by axic** [comment]
@chriseth Rebased.

**Comment by chriseth** [comment]
@axic please squash

**Comment by chriseth** [comment]
(sorry)

**Comment by axic** [comment]
@chriseth done

**Review by chriseth — APPROVED** [review]

**Pull request merged**